### DISCUSSION TITLE
Edit-Thema in Portal-App

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#44] [General]
Ich habe im CMS ein neues Editing-Thema angelegt und wollte dieses dann für eine Portal-App verwenden - im App-Builder unter 'Edit-Thema' wurde das Editing-Thema aber nicht angezeigt.

Ich habe dann stattdessen ein Editing-xml im Verzeichnis erstellt, dann war das Edit-Thema im App-Builder vorhanden und hat funktioniert.

**Comment by gis-eni** [comment]
Kann sein, dass "CMS" Editthemen für Apps generell noch implementiert sind :/

Um welches "App-Template" handelt es sich? Welche App willst du erstellen?

**Reply by thomaskofler** [reply]
Das war WebGIS-Collector-Pro

**Reply by gis-eni** [reply]
Ich hab das bei mir probiert, da kommen auch Editthemen aus dem CMS vor.

Es gibt zwei Möglichkeiten, warum ein Editthema im AppBuilder nicht sichtbar ist:

* Berechtigung: Wenn du das Editthema im CMS berechtigst, muss auch dein ``subscriber::...`` User berechtigt sein.
* Editthema muss über ``Edit-Service`` verfügbar sein: Themen, die in einer AppBuilder App editiert werden, müssen für das ``Edit-Service`` freigeschalten sein:

![image](https://user-images.githubusercontent.com/62870568/222163677-4cbfd4f9-5803-41f5-bb58-e59d1c0c1ec5.png)

Die Option besagt quasi, ob ein Thema auch von einer App editiert werden darf.

**Reply by thomaskofler** [reply]
Ah danke, ich habe das Häckchen beim Edit-Server nicht aktiviert.
